networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,83 @@ These release notes track the development of the Network Observability Operator
+
+* https://errata.devel.redhat.com/advisory/116729
+ https://access.redhat.com/errata/RHSA-2023:1817[RHSA-2023:1817 Network Observability Operator 1.4.0]
+
+[id="network-observability-channel-deprecation-1.4"]
+=== Channel Deprecation
+You must switch your channel from `v1.0.x` to `stable` to receive the latest Operator updates. The `v1.0.x` channel is now deprecated.
+
+[id="network-observability-operator-1.4.0-features-enhancements"]
+=== New features and enhancements
+
+[id="network-observability-without-loki-1.4"]
+==== Network Observability without Loki
+You can now install the Network Observability Operator without Loki and only export flows to KAFKA or IPFIX. For more information, see xref:../../networking/network_observability/installing-operators.adoc#network-observability-without-loki_network_observability[Network Observability without Loki]. 
+
+[id="network-observability-enhanced-configuration-and-ui-1.4"]
+==== Notable enhancements
+The 1.4 release of the Network Observability Operator adds improvements and new capabilities to the {product-title} web console plugin and the Operator configuration. 
+
+This release introduces the following web console enhancements:
+
+* In the *Query Options*, the *Duplicate flows* checkbox is added to choose whether or not to show duplicated flows.
+* You can now filter source and destination traffic with image:arrow-up-long-solid.png[,10] *One-way*, image:arrow-up-long-solid.png[,10] image:arrow-down-long-solid.png[,10] *Back-and-forth*, and *Swap* filters. 
+* The Network Observability metrics dashboards in *Observe* -> *Dashboard* -> *NetObserv* and *NetObserv / Health* are modified as follows:
+** The *NetObserv* dashboard shows top bytes, packets sent, packets received per nodes, namespaces, workload, and pods. Flow graphs are removed from this dashboard.
+** The *NetObserv / Health* dashboard shows flows overhead as well as top flow rates per nodes, namespaces, workload, and pods.
+** Infrastructure and Application metrics are shown in a split-view for namespace, pod, and workloads. 
+  
+
+For more information, see xref:../../networking/network_observability/observing-network-traffic.adoc#network-observability-quickfilter_nw-observe-network-traffic[Configuring Quick Filters] and xref:../../networking/network_observability/network-observability-overview.adoc#network-observability-dashboards[Network Observability metrics].
+ 
+This release introduces the following configuration enhancements:
+
+* The `spec.processor.clusterName` parameter is added so that the name of the cluster appears in the flows data. This is useful in a multi-cluster context. When using {product-title}, leave empty to make it automatically determined.
+* You now have the option to specify different namespaces for all Network Observability components using the `spec.loki.tls.caCert.namespace` and `spec.exporters.kafka.tls.caCert.namespace` fields.
```

Comment:
there's actually more places than loki.tls.caCert and exporters.kafka.tls.caCert, it seems an overkill to list them all, what about saying something like "You now have the option to specify different namespaces for any configured ConfigMap or Secret reference, such as in certificates configuration" ?

Reply:
Okay staying more general here makes sense. I was also thinking that the UI enhancements section may be overkill.